generate/main.go

```diff
@@ -0,0 +1,205 @@
+	tagsBytes, _ := get("https://api.github.com/repos/grafeas/grafeas/tags")
+	var tags []map[string]interface{}
+	json.Unmarshal(tagsBytes, &tags)
+	return tags[0]["name"].(string)
+}
+
+func swaggerCompatibility(jsonInput string) string {
+	re := regexp.MustCompile(`\/{.*(=.*)}`)
+	jsonOutput := re.ReplaceAllStringFunc(jsonInput, func(substringMatch string) string {
+		return strings.Split(substringMatch, "=")[0] + "}"
+	})
+	return jsonOutput
+}
+
+func get(url string) ([]byte, error) {
+	var responseData []byte
+	resp, err := http.Get(url)
+	if err != nil {
+		return responseData, err
+	}
+	defer resp.Body.Close()
+
+	responseData, err = ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return responseData, err
+	}
+	return responseData, err
+}
+
+func downloadCompatibleSwaggerSpec(url string, filename string) error {
+	if filename == "" {
+		filename = path.Base(url)
+	}
+	responseBytes, err := get(url)
+	if err != nil {
+		return err
+	}
+
+	swaggerString := swaggerCompatibility(string(responseBytes))
+
+	f, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	responseReader := bytes.NewReader([]byte(swaggerString))
+
+	_, err = io.Copy(f, responseReader)
+	return err
+}
+
+func downloadFile(url string, filename string) error {
+	if filename == "" {
+		filename = path.Base(url)
+	}
+	responseBytes, err := get(url)
+	if err != nil {
+		return err
+	}
+
+	f, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	responseReader := bytes.NewReader(responseBytes)
+
+	_, err = io.Copy(f, responseReader)
+	return err
+}
+
+type swaggerCodegenConfig struct {
+	language        string
+	inputSpec       string
+	outputDirectory string
+	configFile      string
+	jar             string
+}
+
+func stringOrDefault(currentValue string, defaultValue string) string {
+	if currentValue == "" {
+		return defaultValue
+	}
+	return currentValue
+}
+
+func (c *swaggerCodegenConfig) generate() {
+	c.language = stringOrDefault(c.language, "go")
+	c.jar = stringOrDefault(c.jar, "./swagger-codegen-cli.jar")
+	c.configFile = stringOrDefault(c.configFile, "./config.go.json")
+	c.outputDirectory = stringOrDefault(c.outputDirectory, ServerVersion)
+	args := []string{"-jar", c.jar, "generate", "-i", c.inputSpec, "-l", c.language, "-o", c.outputDirectory, "-c", c.configFile}
+	log.Println("[CMD] java " + strings.Join(args, " "))
+	cmd := exec.Command("java", args...)
+
+	stderr, _ := cmd.StderrPipe()
+	cmd.Start()
+
+	scanner := bufio.NewScanner(stderr)
+	scanner.Split(bufio.ScanLines)
+	for scanner.Scan() {
+		m := scanner.Text()
+		fmt.Println(m)
+	}
+	cmd.Wait()
+}
+
+func downloadSwaggerCodegenCli(version string) {
+	downloadURL := fmt.Sprintf("https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/%s/swagger-codegen-cli-%s.jar", version, version)
+	downloadFile(downloadURL, "swagger-codegen-cli.jar")
+}
+
+func getSwaggerNames() []string {
+	return []string{"grafeas.swagger.json", "project.swagger.json"}
```

Comment:
Pulling in _both_ swagger specs to have access to the `ProjectsAPI` in addition to the `GrafeasV1Beta1Api`